instance/models.py

```diff
@@ -311,11 +311,12 @@ def do_something_with_a_vanished_popit_api_instance(self, popolo_source):
         pass
 
     def _load_persons_from_popolo_json(self, popolo_source):
-        success_relating_people, error = self.relate_with_persons_from_popolo_json(popolo_source)
         record, created = WriteitInstancePopitInstanceRecord.objects.get_or_create(
             writeitinstance=self,
             popolo_source=popolo_source
             )
+        record.set_status('inprogress')
+        success_relating_people, error = self.relate_with_persons_from_popolo_json(popolo_source)
```

Comment:
Might be nice to see an explanation in the commit message of why the order of this has been moved.